.gitignore

```diff
@@ -5,11 +5,22 @@
 /node_modules
 /composer.phar
 behat.yml
-/var/
+/var/*
+!var/cache/
+/var/cache/*
 !var/cache/.gitkeep
```

Comment:
There is no need to add
```
!var/cache/
/var/cache/*
```
as we have .gitkeep inside this folder that already in `.gitignore`, so folder will not be deleted after git init. 
Same for `logs`, `attachments` and` import_export` folders

Reply:
@anyt if you have the following rules the var/cache/.gitkeep file will not be added to a new repository:
```
/var/* 
!var/cache/.gitkeep
```
With or without the `*` above makes no difference

Hopefully I am not doing something silly, but this post seems to explain it: https://stackoverflow.com/a/5534865/358202

Reply:
I guess if you know which folders you want to ignore in `var` then we could have far less rules.  i.e if you only want to ignore cache, logs, attachments and import_export then you only need 8 rules in pairs like:
```
/var/cache/*
 !var/cache/.gitkeep
```
That would save a rule per directory compared to my patch

Reply:
@anyt did you see my above comments ^?